File: src/ios/SpotifySDKPlugin.swift
@objc(SpotifySDKPlugin) class SpotifySDKPlugin: CDVPlugin {
    @objc(runAuth:) func runAuth(command: CDVInvokedUrlCommand) {
        var result = CDVPluginResult(status: CDVCommandStatus_ERROR, messageAs: "The Plugin Failed");
        let args = command.arguments;
        if(args != nil) {
            result = CDVPluginResult(status: CDVCommandStatus_OK, messageAs: "Success")
            var config = STPConfiguration(
            clientID: args[0],
            redirectURL: args[1])
        }
        
        self.commandDelegate!.send(result, callbackId: command.callbackId);
    }
}

File: www/SpotifySDKPlugin.js
var exec = require('cordova/exec');

var PLUGIN_NAME = "SpotifySDKPlugin";

var SpotifySDKPlugin = function() {}

SpotifySDKPlugin.runAuth = function(clientId, redirectURL, onSuccess, onError) {
    exec(onSuccess, onError, PLUGIN_NAME, "runAuth", [clientId, redirectURL]);
};

module.exports = SpotifySDKPlugin;